build.xml

```diff
@@ -23,6 +23,10 @@
     value="com.google.enterprise.adaptor.sharepoint.SharePointAdaptor"/>
   <property name="adaptor2.class"
     value="com.google.enterprise.adaptor.sharepoint.SharePointUserProfileAdaptor"/>
+  <property name="adaptor.title"
+    value="Google Adaptor for SharePoint"/>
```

Comment:
While you're doing this, you could also update these titles to be correct:

Google Search Appliance Connector for SharePoint
Google Search Appliance Connector for SharePoint User Profiles